### PR TITLE
Added Elixir Egg

### LINKED ISSUE #1988: [Egg Request]: Elixir
### Does this expand an already existing service

No

### Link to a game or software

https://elixir-lang.org/

### Links for the download

https://github.com/elixir-lang/elixir

### Links for the install docs

https://elixir-lang.org/install.html

### PR DESCRIPTION
**Pull request by avoxmr**
# Description
Added Elixir Egg.
Resolves #1988 

## Checklist for all submissions

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [X] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

* [X] You verify that the start command applied does not use a shell script
  * [X] If some script is needed then it is part of a current yolk or a PR to add one
* [X] The egg was exported from the panel

## New egg Submissions

1. [X] Does your submission pass tests (server is connectable)?
2. [X] Does your egg use a custom docker image?
    * [X] Have you tried to use a generic image?
    * [X] Did you PR the necessary changes to make it work?
3. [X] Have you added the egg to the main README.md and any other README files in subdirectories of the egg (e.g /game_eggs) according to the alphabetical order?
4. [X] Have you added a unique README.md for the egg you are adding according to the alphabetical order?
5. [X] You verify that the start command applied does not use a shell script
    * [X] If some script is needed then it is part of a current yolk or a PR to add one
6. [X] The egg was exported from the panel

### CODE REVIEW

File: generic/elixir/egg-elixir.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            "container": "debian:buster-slim",
```
Comment (QuintenQVD0) [inline generic/elixir/egg-elixir.json:30]: Use bullseye-slim

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Comment by QuintenQVD0** [comment]
You can not directly edit the .json file! All changes must be made in a panel, then exported 1nd fully replaced

**Comment by avoxmr** [comment]
Ah i see, thanks for letting me know.

**Comment by avoxmr** [comment]
I have replaced the old egg with a new egg that i have exported from the panel.

**Review by QuintenQVD0 — APPROVED** [review]
LGTM!

**Pull request merged**